### DISCUSSION TITLE
How i get student github id

### DISCUSSION THREAD

**Discussion by mohitsig** [community/community#137590] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

How i get student github id

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @SamratSuraj0 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
